pages/common/gfortran.md

```diff
@@ -0,0 +1,24 @@
+# gfortran
+
+> Preprocess and compile Fortran source files, then assemble and link them together.
+> More information: <https://gcc.gnu.org/wiki/GFortran>.
+
+- Compile multiple source files into executable:
```

Comment:
```suggestion
- Compile multiple source files into an executable:
```

Reply:
Good point, then you may want to change it here, too:
https://github.com/tldr-pages/tldr/blob/main/pages/common/gcc.md

Reply:
> Good point, then you may want to change it here, too: https://github.com/tldr-pages/tldr/blob/main/pages/common/gcc.md

Thanks, will make the change over there too.